src/sections/header.html

```diff
@@ -1,17 +1,31 @@
-<header class="font-outfit py-4">
-    <nav class="fixed right-10 left-10 ">
-        <div class="flex justify-between">
-            <div>
-                <a href="#"><p class="text-2xl">EN</p></a>
-                <a href="#"><p class="text-2xl">DEN</p></a>
+<header class="font-outfit lg:py-8  sm:py-8">
+    <nav class="fixed md:right-10 md:left-10 right-5 left-5 top-4">
+        <div class="flex justify-between ">
+            <div class="">
+                <a href="#"><p class="md:text-2xl text-xl">EN</p></a>
+                <a href="#"><p class="md:text-2xl text-xl">DEN</p></a>
             </div>
-            <h1 class="text-5xl font-extrabold flex items-center ">Baumeister</h1>
-            <ul class="text-2xl">
+            <h1 class="md:text-5xl md:font-extrabold flex items-center text-4xl font-semibold relative">Baumeister</h1>
+            <img src="src/assets/images/white-shape.svg" alt="white-shape" class="md:hidden block absolute -z-10 left-16 w-7/12">
+            <ul class="hidden lg:block lg:text-2xl">
                 <a href="#about"><li>About</li></a>
                 <a href="#services"><li>Services</li></a>
                 <a href="#reference"><li>References</li></a>
                 <a href="../../contact.html"><li>Contact</li></a>
             </ul>
+            <!-- nav hamburger menu -->
+            <!-- <div id="hamburger" class="hidden">
+                <ul class="flex flex-col gap-4">
+                    <a href="#"><li>About</li></a>
+                    <a href="#"><li>Services</li></a>
+                    <a href="#"><li>References</li></a>
+                    <a href="#"><li>Contact</li></a>
+                </ul>
+            </div> -->
+            <div class="lg:hidden flex items-center  w-[3rem] md:w-[5rem]" id="menu">
```

Comment:
`w-[3rem]` It would be better to use default TailwindCSS values than arbitrary whenever possible. Like here, tailwind values would be a better fit.